xc/xenctrlext.mli

```diff
@@ -71,3 +71,11 @@ external numainfo: handle -> numainfo = "stub_xenctrlext_numainfo"
 external cputopoinfo: handle -> cputopo array = "stub_xenctrlext_cputopoinfo"
 
 external xc_get_msr_arch_caps : handle -> int64 = "stub_xenctrlext_get_msr_arch_caps"
+
+external xc_cpu_policy_get_system: handle -> int -> int64 array = "stub_xc_cpu_policy_get_system"
```

Comment:
see above, if you want to pass it directly use bigarrays (and use the right macro to gain access to the pointer you can pass to C)